spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -78,8 +78,8 @@ public Span(Span current, Span savedSpan) {
 		this.remote = current.isRemote();
 		this.exportable = current.isExportable();
 		this.processId = current.getProcessId();
-		this.tags.putAll(current.tags());
-		this.logs.addAll(current.logs());
+		this.tags = current.tags;
```

Comment:
I've noticed that I've forgotten to add a test - good catch. I'll also add a javadoc to the constructor